app/Http/Requests/ShareDriveFileRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Requests;
+
+use App\Enums\DriveFilePermissionEnum;
+use App\Enums\DriveFileShareExpiresAtEnum;
+use Illuminate\Foundation\Http\FormRequest;
+
+class ShareDriveFileRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
```

Comment:
instead of this in controller : `abort_if($file->user_id !== $user->id, Response::HTTP_FORBIDDEN)`

we can check it in the form request